react/features/recording/components/Recording/StartRecordingDialogContent.js

```diff
@@ -452,7 +452,7 @@ class StartRecordingDialogContent extends Component<Props> {
             );
         }
 
-        if (this.props.fileRecordingsServiceEnabled) {
+        if (this.props.fileRecordingsServiceEnabled || this._localRecordingAvailable) {
```

Comment:
This change looks odd. The value this switch represents is Dropbox, so why are we adding local recordings here?

Reply:
So this is how it works: if dropbox is the only option show a button instead of a toggle. If the file service is enabled show toggle instead of button. And this change is if local recording is available show toggle instead of button

Reply:
Why not have consistent controls?

Reply:
Because if you only have dropbox and display a toggle (which will be on) you can't get to the sign in page. When multiple services are enabled the login is opened when you toggle Dropbox on (which you can't do if it's already on). So a button is displayed instead

Reply:
I didn't create this behavior. I'm just fixing it to work with local rec :D

Reply:
I see.